packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -263,90 +266,117 @@ export default function NewCommentModal({
     }
   }, []);
 
+  const isPreview = activeTab === 'Preview';
+  const { data: previewContent } = useQuery<{ preview: string }>(
+    input,
+    () =>
+      request(`${apiUrl}/graphql`, PREVIEW_COMMENT_MUTATION, {
+        content: input,
+      }),
+    { enabled: isPreview && input?.length > 0 },
+  );
+
   return (
     <ResponsiveModal
       {...{ contentRef: modalRef, onRequestClose: confirmClose, ...props }}
+      padding={false}
+      contentClassName="p-2"
     >
-      <article
-        className={`flex flex-col items-stretch ${commentBoxClassNames}`}
+      <TabContainer
```

Comment:
Maybe we should move the tabs to their own component, this is getting quite big now.
Let me know what you think

Reply:
Great idea! Let me extract this one out.

Reply:
Great idea! Let me work on that 🚢